### PR TITLE
BEI-14: Register users

### PR DESCRIPTION
**Pull request by hizikex**
#### WHY THIS PR?
- There is need for users to register on this paltform.

#### WHAT DOES THIS PR DO?
-It allows users to register with at least their username, email and password.

HOW CAN THIS PR BE TESTED?
-By not passing a value in either of the field mentioned above, a user will not be able to register on foodies_haven.

THINGS YOU MAY WANT THE REVIEWER TO NOTE.
N/A

### CODE REVIEW

File: src/core/helpers/cloudinary.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {v2 as cloudinary} from 'cloudinary';
+
```
Comment (LucianaDera) [inline src/core/helpers/cloudinary.ts:2]: are we making use of cloudinary?

**Review by LucianaDera — CHANGES_REQUESTED** [review]

File: src/core/helpers/cloudinary.ts (hunk 1 above)
Comment (RICHYDOS) [reply src/core/helpers/cloudinary.ts:2]: @hizikex please respond to this

**Review by RICHYDOS — APPROVED** [review]

**Review by LucianaDera — APPROVED** [review]

File: src/core/repositories/User.ts [hunk 2]
```diff
@@ -2,6 +2,9 @@ import { User } from '../models/users';
 import BaseRepository from './Base';
 
 export default class UserRepository extends BaseRepository<User> {
+  findByPk(userId: number) {
+    throw new Error('Method not implemented.');
+  }
```
Comment (RICHYDOS) [inline src/core/repositories/User.ts:7]: What does this do?

**Review by RICHYDOS — CHANGES_REQUESTED** [review]

File: src/core/repositories/User.ts (hunk 2 above)
Comment (hizikex) [reply src/core/repositories/User.ts:7]: Is not supposed to be here.

**Review by RICHYDOS — APPROVED** [review]

**Pull request merged**